packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -2556,7 +2556,7 @@ void main() {
     await gesture.down(textOffsetToPosition(tester, 5));
     await tester.pump();
     await gesture.up();
-    await tester.pumpAndSettle();
+    await tester.pumpAndSettle(kDoubleTapTimeout);
```

Comment:
Ah I guess some tests that previously tapped very soon after a double tap might behave differently now.  We should watch out for this breaking Google tests or anything else.  Though it looks like Google tests are currently passing 👍

Reply:
Yeah, how it worked previously is that the double tap timer on `TextSelectionGestureDetector` would reset the count to 0 after every double tap. So some test behavior changed a bit, but google tests look good so far.